src/sid/countdowns.py

```diff
@@ -42,7 +42,7 @@
     "cd_knows_immune_false": {"changes": {"knows_immune": False}},
     "cd_knows_infectious_false": {"changes": {"knows_infectious": False}},
     "cd_ever_infected": {},
-    "cd_is_immune_by_vaccine": {"changes": {"immune": True}},
+    "cd_is_immune_by_vaccine": {},
```

Comment:
This is now empty since the change depends on the ``states`` and ``params`` data frame. The change is applie din ``update_states`` after a check that ``cd_is_immune_by_vaccine == 0``.

Reply:
If we model waning immunity we could just as well model an increase in the immunity level during the first 2 weeks. Do we need this countdown then?